x-pack/plugins/cases/server/services/user_actions/operations/create.ts

```diff
@@ -0,0 +1,428 @@
+/*
```

Comment:
Shouldn't be any new functionality in here just the class structure and copied methods from the user action service.